airbyte-db/lib/src/main/java/io/airbyte/db/mongodb/MongoUtils.java

```diff
@@ -70,6 +77,24 @@ public static JsonNode toJsonNode(final Document document, final List<String> co
     return objectNode;
   }
 
+  public static Object getBsonValue(BsonType type, String value) {
+    try {
+      return switch (type) {
+        case INT32 -> new BsonInt32(Integer.parseInt(value));
+        case INT64 -> new BsonInt64(Long.parseLong(value));
+        case DOUBLE -> new BsonDouble(Double.parseDouble(value));
+        case DECIMAL128 -> Decimal128.parse(value);
+        case TIMESTAMP -> new BsonTimestamp(Long.parseLong(value));
+        case DATE_TIME -> new BsonDateTime(new DateTime(value).getValue());
+        case OBJECT_ID -> new ObjectId(value);
+        default -> null;
```

Comment:
instead of default null why don't we return string? 

Also I don't see a mapping from string or object, how come?

Reply:
I've changed to String, thanks! Also added String and Symbol.
This mapping needed only for cursor field, since it couldn't be object or array, it isn't present here.